<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>插件市场</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(250, 250, 250);
	border:none;
}
QGroupBox{
	color: rgb(0, 0, 0);
}
QScrollBar:vertical{
	border: none;
	width:8px;
	margin: 0 0 0 0;
	background: transparent;
}
QScrollBar::handle:vertical{
	background: rgb(200, 200, 200);
	border-radius:4px;
}
QScrollBar::handle:vertical:hover{
	background: rgb(160, 160, 160);
	border-radius:4px;
}
QScrollBar::handle:vertical:pressed{
	background: rgb(5, 160, 255);
	border-radius:4px;
}
QScrollBar::add-page,QScrollBar::sub-page:vertical{
	background: transparent;
}
QScrollBar::add-line,QScrollBar::sub-line:vertical{
	background: transparent;
}
QLineEdit{
	border:1px solid rgb(200,200,200);
	border-radius:5px;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	selection-background-color: rgb(0, 0, 0);
	selection-color: rgb(255, 255, 255);
}
QPushButton{
	color: rgb(0, 0, 0);
	border-radius:5px;
	background-color: rgb(255, 255, 245);
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	border-radius:5px;
	background-color: rgb(255, 255, 252);
}
QListWidget{
	background:transparent;
	border:none;
}
QListWidget::item:hover{
	background-color:rgb(220,220,220);
	border-radius:5px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="3,1,1">
   <item row="0" column="0">
    <widget class="QLineEdit" name="ipLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="portLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="findButton">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>搜索插件</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/resource/find.png</normaloff>:/resource/find.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="settingsButton">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>网络配置</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/resource/settings.png</normaloff>:/resource/settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
